ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc

```diff
@@ -111,16 +120,43 @@ TorchCore::~TorchCore ()
   g_free (model_path);
 }
 
+/**
+ * @brief	Set the accelerator for the pytorch
+ */
+void
+TorchCore::setAccelerator (const char *accelerators)
+{
+  use_gpu = TRUE;
+  accelerator = parse_accl_hw (accelerators, torch_accl_support);
+  if (accelerator == ACCL_NONE)
+    goto use_gpu_ini;
+  if ((accelerator & (ACCL_CPU | ACCL_DEFAULT | ACCL_AUTO)) != 0)
```

Comment:
ACCL_AUTO means no GPU?

Reply:
This is becasuse the pytorch package provided with nnstreamer launchpad does not support GPU. Should I update ACCL_AUTO to imply GPU?

Reply:
It should use GPU if it can use GPU it won't use GPU if it cannot (or is not recommended to) use GPU. You need not to disable it with AUTO.

Reply:
Updated ACCL_AUTO to use GPU.